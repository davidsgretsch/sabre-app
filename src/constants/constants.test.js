import * as constants from '.';

describe('constants', () => {
  it('should match snapshot', () => {
    expect(constants).toMatchSnapshot();
  });
});
