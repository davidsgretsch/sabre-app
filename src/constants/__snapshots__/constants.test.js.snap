// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should match snapshot 1`] = `
Object {
  "API_URL": "https://football-players-b31f2.firebaseio.com/players.json",
  "PLAYER_POSITIONS": Array [
    Object {
      "value": "Attacking Midfield",
    },
    Object {
      "value": "Central Midfield",
    },
    Object {
      "value": "Centre-Back",
    },
    Object {
      "value": "Centre-Forward",
    },
    Object {
      "value": "Defensive Midfield",
    },
    Object {
      "value": "Keeper",
    },
    Object {
      "value": "Left Midfield",
    },
    Object {
      "value": "Left Wing",
    },
    Object {
      "value": "Left-Back",
    },
    Object {
      "value": "Right-Back",
    },
  ],
}
`;
